client-app/builder-preview/pages/pages.vue

```diff
@@ -2,7 +2,9 @@
 <template>
   <div class="max-w-screen-2xl px-5 md:px-12 mx-auto">
     <div class="block py-8" v-for="block in template.content" :key="block.id">
-      <component :is="block.type" :model="block" />
+      <div v-if="!block.hidden">
```

Comment:
Please, move this check to the previous line. Now you'll have redundant paddings for empty blocks, cause line 4 with its classes will be rendered for every (even hidden) block.

Reply:
fixed